prov/efa/src/rxr/rxr_read.h

```diff
@@ -117,7 +117,7 @@ int rxr_read_init_iov(struct rxr_ep *ep,
 
 int rxr_read_post(struct rxr_ep *ep, struct rxr_read_entry *read_entry);
 
-int rxr_read_post_remote_read_or_queue(struct rxr_ep *ep, int entry_type, void *x_entry);
+int rxr_read_post_remote_read_or_queue(struct rxr_ep *ep, struct rxr_op_entry *x_entry);
 
 int rxr_read_post_local_read_or_queue(struct rxr_ep *ep,
 				      struct rxr_rx_entry *rx_entry,
```

Comment:
So we add macros for `rxr_rx_entry` and `rxr_tx_entry` to avoid changing too many variable name like this?

Reply:
Yes.

I was planning to make the transition increamently. New code will be using "struct rxr_op_entry".

If you prefer, I can also do a systematic change via a `sed` command.

Reply:
Thanks, I am ok with making the change incrementally.